### DISCUSSION TITLE
AWS S3 - One Bucket Won't Display

### DISCUSSION THREAD

**Discussion by mrriehl** [iterate-ch/cyberduck#15463] [Q&A]
Hi - We have several AWS S3 buckets and all but one bucket is visible using Mountain Duck on macOS. The top-level bucket folder appears, however, when we click it to see the contents, it appears to be empty. The bucket has content and is visible from the AWS CLI on the same laptop.

The one bucket we can't see is ~5 TB with ~250k files. Could this be an issue?

This was also the only bucket that had Bucket Versioning enabled. However, we disabled it and still didn't have any luck.

Any suggestions on how to track this down?

**Comment by dkocher** [comment]
This might be a performance issue in Finder.app accessing the folder with many files. What output do you get from `ls` in Terminal.app?

**Reply by mrriehl** [reply]
I see where it is mounted, if I cd into that folder and run `ls`, I get:

`ls: fts_read: RPC struct is bad`

However, the contents of other folders that are visible with Mountain Duck appear as expected.
